tests/Microsoft.Identity.Test.LabInfrastructure/KeyVaultSecretsProvider.cs

```diff
@@ -119,54 +119,78 @@ public KeyVaultSecretsProvider()
 
         public SecretBundle GetSecret(string secretUrl)
         {
-            return _keyVaultClient.GetSecretAsync(secretUrl).GetAwaiter().GetResult();
+            return KeyVaultClient.GetSecretAsync(secretUrl).GetAwaiter().GetResult();
         }
 
         private async Task<string> AuthenticationCallbackAsync(string authority, string resource, string scope)
```

Comment:
nit: scope ?

Reply:
this method is used as the "AuthenticationCallback" for keyvault. it requires the scope parameter to be present.
https://docs.microsoft.com/en-us/dotnet/api/microsoft.azure.keyvault.keyvaultclient.authenticationcallback?view=azure-dotnet